.github/workflows/javascript-test.yml

```diff
@@ -0,0 +1,19 @@
+name: Continuous testing
+
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use node 18
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
```

Comment:
18 is already LTS, huh
guess we can start upgrading it in all the other places.

Reply:
Yup! Technically speaking we can even use the [native node test runner](https://nodejs.org/de/blog/announcements/v18-release-announce/#test-runner-module-experimental) for our projects but I needed some substitution library.